aten/src/ATen/detail/AcceleratorHooksInterface.h

```diff
@@ -48,7 +46,7 @@ struct TORCH_API AcceleratorHooksInterface {
     return 0;
   }
 
-  virtual void setCurrentDevice(DeviceIndex device) const {
+  virtual void setCurrentDevice(DeviceIndex device [[maybe_unused]]) const {
```

Comment:
Isn't the proper way to handle unused args just to comment out the lname with '/* */'. That way it can be bound to anything and is clearly unused?

Reply:
Makes sense. Let me rewrite them.

Reply:
Fixed